src/common/personalizedDigest.ts

```diff
@@ -268,13 +264,8 @@ export const schedulePersonalizedDigestSubscriptions = async ({
     emailBatchId: string;
   }) => Promise<void>;
 }) => {
-  // TODO: check if there's an alternative in CIO
-  const emailBatchId = await createEmailBatchId();
-
-  if (!emailBatchId) {
-    throw new Error('failed to create email batch id');
-  }
-
+  // Keep email batch id around just in case
+  const emailBatchId = null;
```

Comment:
If we are keeping it maybe generate with `generateUUID`? It will be useful since each message log has it so if any errors we know from which cron it comes from.

Reply:
yes, fixed